ui/src/common/SongInfo.js

```diff
@@ -17,12 +17,24 @@ import inflection from 'inflection'
 import { BitrateField, SizeField } from './index'
 import { MultiLineTextField } from './MultiLineTextField'
 import { makeStyles } from '@material-ui/core/styles'
+import config from '../config'
 
-const useStyles = makeStyles({
-  tableCell: {
-    width: '17.5%',
-  },
-})
+const useStyles = config.enableReplayGain
+  ? makeStyles({
```

Comment:
There's a better (more MUI) way to achieve this: https://github.com/navidrome/navidrome/blob/31c598de075fc7af96be93070056a6666da1b5c3/ui/src/common/SongDatagrid.js#L48